### DISCUSSION TITLE
[Question][Hw1] Minor issue for format

### DISCUSSION THREAD

**Discussion by kaist-kihwan** [prosyslab-classroom/cs348-information-security#669] [Q&A]
Name: Kim Kihwan

I thought we set every plain text and decrypted message are written in lowercase, but dec function of vigenere cipher in hw1 requires uppercase. It makes me submit the homework twice, so I hope this kind of situation does not happen for further homework.

**Comment by yeonhee-ryou** [comment]
Sorry for the confusion.

Search for similar questions before posting questions. duplicated #222
Also, test your codes with the given test cases before submitting them.
